bin/improver-threshold

```diff
@@ -162,8 +162,17 @@ def main():
         # smooth thresholded occurrences over local vicinity
         result_no_collapse_coord = OccurrenceWithinVicinity(
             args.vicinity).process(result_no_collapse_coord)
-        result_no_collapse_coord.rename(
-            result_no_collapse_coord.name() + '_in_vicinity')
+        cube_name = result_no_collapse_coord.name()
```

Comment:
This could be a function that we move somewhere and unit test. What do you think?

Reply:
This CLI already has quite a bit of 'pre-processing' going on and I didn't think too much of adding this change here. However I shouldn't use this as an excuse and I agree that processing should be kept outside of CLI's as far as possible.

So I've moved this into a function within` utilities/cube_metadata.py ` as this seemed the most logical place to put it. Accompanying unit test created.